test/e2e/queue.go

```diff
@@ -418,15 +427,20 @@ var _ = Describe("AppWrapper E2E Test", func() {
 		appwrappersPtr := &appwrappers
 		defer cleanupTestObjectsPtr(context, appwrappersPtr)
 
-		// This should fill up the worker node and most of the master node
-		aw := createDeploymentAWwith550CPU(context, appendRandomString("aw-deployment-2-550cpu"))
+		//This should fill up the worker node and most of the master node
+		//aw := createDeploymentAWwith550CPU(context, appendRandomString("aw-deployment-2-550cpu"))
+		cap := getClusterCapacitycontext(context)
+		resource := cpuDemand(cap, 0.275).String()
+		aw := createGenericDeploymentCustomPodResourcesWithCPUAW(
```

Comment:
What happens if the cluster has many smaller nodes resulting a a high `cap` but inability to schedule AppWrappers becauase they do not fit on the individual nodes? Do we care about that at al in this test case?

Reply:
From a test case perspective, the cluster is assumed to have homogenous nodes and it requests deployments that fit on a node in the cluster in CPU dimension.